src/js/pages/jobs/JobsTable.js

```diff
@@ -96,7 +100,22 @@ class JobsTable extends React.Component {
     });
   }
 
-  sortJobNames(prop, direction) {
+  getCompareFunctionByProp(prop) {
+    switch (prop) {
+      case "name":
+        return (a, b) => a.name.localeCompare(b.name);
+      case "status":
+        return (a, b) =>
+          JobStates[a.status].sortOrder - JobStates[b.status].sortOrder;
+      case "lastRun":
+        return (a, b) =>
+          JobStatus[a.lastRun.status].sortOrder -
+          JobStatus[b.lastRun.status].sortOrder;
+    }
+  }
+
+  jobSortFunction(prop, direction) {
```

Comment:
I named this way to conform to the same naming as [ServiceTableUtil.js](https://github.com/dcos/dcos-ui/blob/master/plugins/services/src/js/utils/ServiceTableUtil.js).